spec/hash_mergers/hash_merger_spec.rb

```diff
@@ -24,9 +24,12 @@
   end
 
   def build_merged_hash(**hash_input)
-    Metalware::HashMergers.merge(**hash_input) do |template_string|
-      template_string
-    end
+    hm = Metalware::HashMergers
+    b = proc { |t| t }
+    OpenStruct.new(
+      config: hm::Config.new.merge(**hash_input, &b),
+      answer: hm::Answer.new(alces).merge(**hash_input, &b)
```

Comment:
`b` here just returns the object it is passed, so you can just replace both uses here with `&:itself` (see [`Object.itself`](http://devdocs.io/ruby~2.4/object#method-i-itself)) and remove defining `b`.

Reply:
Done